doc/USAGE-ospd-scanner

```diff
@@ -0,0 +1,94 @@
+GENERAL USAGE INSTRUCTIONS for OSPD-BASED SCANNERS
+--------------------------------------------------
+
+This is a general description about how to use a
+ospd-based scanner wrapper implementation.
+
+The actual scanner implementation has a individual
+README instruction for anything that goes beyond
+this general guide.
+
+In the following description replace "scanner" by the name
+of the actual OSPD scanner.
+
+See the INSTALL of your ospd-based scanner and the general
+instructions in file ospd/doc/INSTALL-ospd-scanner on how to
+hand over full control to the Greenbone Vulnerability Manager.
+
+This usage guide explains how to use a OSP scanner independently
+from Greenbone Vulnerability Manager, for example when developing
+a new ospd-based scanner or for testing purposes.
+
+
+Open Scanner Protocol
+---------------------
+
+Using a ospd-based scanner means to use the Open Scanner Protocol
+(OSP). This is what Greenbone Vulnerability Manager does.
+See the ospd module for the original specificatoin (ospd/doc/OSP.xml).
+
+There are also online version, for example: 
+
+  http://docs.greenbone.net/API/OSP/osp-1.0.html
+
+
+gvm-tools
+---------
+
+The gvm-tools help to make accessing the OSP interface easier.
+It can be optained from
+
+  https://github.com/greenbone/gvm-tools
+
+This module provide the comands gvm-cli and gvm-pyshell.
+
+
+Starting a ospd-based scanner
+-----------------------------
+
+All ospd-based scanners share a set of command-line options such as
+--help, --bind-address, --port, --key-file, --timeout etc.
+
+For example to see the command line options you can run
+
+$ ospd-scanner --help
+
+
+To run a test instance of ospd-scanner on unix socket:
+
+$ ospd-scanner -u <prefix>/var/run/ospd-scanner.sock &
+
+To run a test instance of ospd-scanner on local port 1234:
+
+$ ospd-scanner -b 127.0.0.1 -p 1234 &
+
+Add "--log-level=DEBUG" to enable maximum debugging output.
+
+Parameter for --log-level can be one of debug, info, warnings, error, or
+critical (in order of priority).
+
+
+Controlling a OSP scanner
+-------------------------
+
+You can use command line tools of "gvm-tools" module to send to and
+receive information from the scanners.
+
+Get a description of the interface:
+
+$ gvm-cli socket --sockpath <prefix>/var/run/ospd-scanner.sock --xml "<help/>"
+
+or
+
+$ gvm-cli --use-certs -p 1234 -i --xml="<help/>"
```

Comment:
This command is not a valid command for `gvm-cli`. Please consider adjusting or removing it.